index.html

```diff
@@ -41,9 +47,10 @@
         </div>
         <div class="resume">
             <div class="resume-items">
+                <h1>Experience</h1>
```

Comment:
Since I suggested making the cover text an `h1` and there's an unspoken rule of having only one `h1` per page (even though it's totally valid to have multiple), I'd suggest using `h2` for this.

```suggestion
                <h2>Experience</h2>
```